src/js/navigation.js

```diff
@@ -12,6 +12,13 @@
   const firstFocusableEl = focusableNavElements[0];
   const lastFocusableEl = focusableNavElements[focusableNavElements.length - 1];
 
+  const isDefaultMobileNav = body.classList.contains('is-always-mobile-nav');
+  const observerConfig = {
+    root: document.querySelector('.site-header'),
+    rootMargin: '0px -75% 0px 0px'
```

Comment:
What does the `-75%` refer to?

Reply:
Yeah, can see how this isn't very intuitive. The root has to contain the element that we want to observe (the primary nav) and in this case also the area that we want to track intersection with (site-branding.) So adjusting the root margin using that value is intended to essentially draw a box around header__left and site-branding. This screenshot probably illustrates the concept better than my words:

![Screenshot at Mar 11 09-21-28](https://user-images.githubusercontent.com/889478/76427196-02a8d380-637a-11ea-9b1b-a27d84c2cc19.png)

-75% percent feels like a pretty loose way to define that, but in testing this I found it to work well. It would be possible to calculate the actual value, but the problem with that is that we'd have to recalculate it on resize and then update the observer, which I'm not even really sure how to do.